debian/changelog

```diff
@@ -1,3 +1,12 @@
+eos-application-tools (1.0.4) eos; urgency=medium
```

Comment:
The CI build failed, and I think it is because of this line.  If you want to bump the version number in the changelog, I think we also have to apply new git labels, which I suppose we can do after merging the PR.

Reply:
Yeah we can push a new tag. We could do that already on the branch I think?

Reply:
If we do it before the merge, it will probably work though it won't exactly correspond to the resulting commit on the master branch.  I'm guessing it would make the actual commit on master appear to be a "+dev1", but I'm not sure if the logic will get confused at all.

Reply:
hmm ok let's tag it after the merge. Will do a fixup later but if you want to amend the patch go ahead.